egs2/TEMPLATE/enh1/enh.sh

```diff
@@ -669,6 +672,21 @@ if ! "${skip_train}"; then
             done
         fi
 
+        # Add the category information at the end of the data path list
```

Comment:
It is a bit redundant I think. Maybe we can do something in the sampler and preprocessor instead of here.

Reply:
I think that will require more changes in more scripts. The current design only needs modifications in `enh.sh`, `espnet2/tasks/enh.py`, and `espnet2/train/preprocessor.py`.